### PR TITLE
Veronika Bogdanovich 051006

### PR DESCRIPTION
**Pull request by VeronikaBogdanovich744**
@WildTiger404

### CODE REVIEW

**Comment by WildTiger404** [comment]
You should add git ignore and delete folders bin and obj

File: BaseFigure.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresLR1
+{
+    internal abstract class BaseFigure //current build
```
Comment (WildTiger404) [inline BaseFigure.cs]: what is the plot of this comment?

File: Rhombus.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresLR1
+{
+    class Rhombus:Quadrangle
+    {
+        private int x1, y1;
```
Comment (WildTiger404) [inline Rhombus.cs]: why do you need those variables, if Quadrangle class has it?

File: Quadrangle.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresLR1
+{
+    class Quadrangle : BaseFigure
+    {
+        private protected int x1, y1;
```
Comment (WildTiger404) [inline Quadrangle.cs:11]: Your code doesn't have an implementation to change coordinates if you need it. Read about get; set; and why do people use it

**Review by WildTiger404 — CHANGES_REQUESTED** [review]

File: BaseFigure.cs (hunk 1 above)
Comment (VeronikaBogdanovich744) [reply BaseFigure.cs]: There is no plot. I've just forgotten to delete it 😅

**Pull request merged**